src/pkg/packager/prepare.go

```diff
@@ -158,18 +158,11 @@ func (p *Packager) findImages(ctx context.Context) (map[string][]string, error)
 		resources := []*unstructured.Unstructured{}
 		matchedImages := map[string]bool{}
 		maybeImages := map[string]bool{}
-		for _, chart := range component.Charts {
+		for _, zarfChart := range component.Charts {
```

Comment:
Q: Why zarfChart here? We're within zarf, so I would expect it to be first class and helmChart to gets prefixed.